applications/fax/src/fax_monitor.erl

```diff
@@ -79,6 +79,9 @@ handle_cast(_Msg, State) ->
 %% @end
 %%------------------------------------------------------------------------------
 -spec handle_info(any(), state()) -> kz_types:handle_info_ret_state(state()).
+handle_info('timeout', #state{pending_migrate='true'}=State) ->
```

Comment:
This seems like the wrong way to go about this.

Why would we need to migrate pending faxes 5s after the `fax_monitor` process starts up, every time the process starts?

Shouldn't this be part of a `kapps_maintenance:migrate` process?

Reply:
Additionally, if this needs to be run, let's set init's return to `{ok, state, 0}` (or more cleanly, `gen_server:cast(self(), 'migrate_pending'), and leave init unchanged, adding the migrate to `handle_cast/2`

Reply:
Because faxes existing in the old format will not be visible via the updated views, so we have to migrate them as fax_worker no longer knows how to convert to tiff, it just assumes tiff now. The migration would typically hit the view, see nothing pending_migration and continue on. 

We also want it to block until the migration has completed, but not block the app from starting so other apps can start up without waiting for this. 

I will look into handling this via handle cast.

Reply:
But it should be a one-time migration, right? As-is, it will run on every start of the fax app.